doc/reference/pipeline_spec.md

```diff
@@ -274,42 +283,43 @@ The default if left unset is "constant=1".
+each worker needs (with allowed SI suffixes (M, K, G, Mi, Ki, Gi, and so on).
+For example, a worker that needs to read a 1GB file into memory might set
+`"memory": "1.2G"` with a little extra for the code to use in addition to the
+file. Workers for this pipeline will be placed on machines with at least
 1.2GB of free memory, and other large workers will be prevented from using it
 (if they also set their `resource_requests`).
 
-The `cpu` field is a number that describes the amount of CPU time (in (cpu
-seconds)/(real seconds) each worker needs. Setting `"cpu": 0.5` indicates that
+The `cpu` field is a number that describes the amount of CPU time in `cpu
+seconds/real seconds` that each worker needs. Setting `"cpu": 0.5` indicates that
 the worker should get 500ms of CPU time per second. Setting `"cpu": 2`
-indicates that the worker should get 2000ms of CPU time per second (i.e. it's
-using 2 CPUs, essentially, though worker threads might spend e.g. 500ms on four
-physical CPUs instead of one second on two physical CPUs).
+indicates that the worker gets 2000ms of CPU time per second. In other words,
+it is using 2 CPUs, though worker threads might spend 500ms on four
+physical CPUs instead of one second on two physical CPUs.
 
 The `disk` field is a string that describes the amount of ephemeral disk space,
-in bytes, each worker needs (with allowed SI suffixes (M, K, G, Mi, Ki, Gi,
-etc).
+in bytes, each worker needs with allowed SI suffixes (M, K, G, Mi, Ki, Gi,
+and so on).
 
 In both cases, the resource requests are not upper bounds. If the worker uses
-more memory than it's requested, it will not (necessarily) be killed.  However,
-if the whole node runs out of memory, Kubernetes will start killing pods that
-have been placed on it and exceeded their memory request, to reclaim memory.
-To prevent your worker getting killed, you must set your `memory` request to
+more memory than it is requested, it does not mean that it will be shut down.
+However, if the whole node runs out of memory, Kubernetes starts deleting
+pods that have been placed on it and exceeded their memory request,
+to reclaim memory.
+To prevent deletion of your worker node, you must set your `memory` request to
 a sufficiently large value. However, if the total memory requested by all
-workers in the system is too large, Kubernetes will be unable to schedule new
-workers (because no machine will have enough unclaimed memory). `cpu` works
+workers in the system is too large, Kubernetes cannot schedule new
+workers because no machine has enough unclaimed memory. `cpu` works
 similarly, but for CPU time.
 
 By default, workers are scheduled with an effective resource request of 0 (to
 avoid scheduling problems that prevent users from being unable to run
-pipelines).  This means that if a node runs out of memory, any such worker
-might be killed.
+pipelines). This means that if a node runs out of memory, any such worker
+might be destroyed.
```

Comment:
killed is actually the correct word here. When a node runs out of memory, the actual error message is "killed" and we call this OOM Killed (out of memory)